test/cypress/fixtures/test.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Name": "Alex Jin",
+  "Birthday": "02.20.1992"
```

Comment:
Do we really need PII for test data?

Reply:
Of course, not. I forgot it. Thanks for your catch.

Reply:
I'm clicking "Unresolve" on this to not forget about it (since the updates haven't yet been pushed)